### PR TITLE
Use symbols for icons in SwiftUI

### PR DESCRIPTION
**Pull request by cobbal**
[ignore-commit-lint]

### CODE REVIEW

**Comment by inst-danger** [comment]
<details><summary>Student.ipa</summary>
  <a href="https://app.bitrise.io/artifact/49432444/p/066a118b26928f4209e57368f59fde43">
    
  </a>
</details>
<details><summary>Teacher.ipa</summary>
  <a href="https://app.bitrise.io/artifact/49432453/p/e32652d3b3d8e1a39ce2c5b971f40948">
    
  </a>
</details>
<details><summary>Parent.ipa</summary>
  <a href="https://app.bitrise.io/artifact/49432426/p/966c13e6ad518bcde3a3cee471d61681">
    
  </a>
</details>

**Comment by inst-danger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  One or more files are below the minimum test coverage 50%
  </td>
    </tr>
  
<tr>
      <td>:no_entry_sign:</td>
      <td>

  Please add a reference to a jira ticket. For example: `refs: MBL-10023`
  </td>
    </tr>
  
<tr>
      <td>:no_entry_sign:</td>
      <td>

  Please add which apps this change affects. Example: `affects: Teacher, Student` or `affects: none`
  </td>
    </tr>
  
<tr>
      <td>:no_entry_sign:</td>
      <td>

  Please add a release note. If no release note is wanted, use `none`. Example: `release note: Fixed a bug that prevented users from enjoying the app.`
  </td>
    </tr>
  </tbody>
</table>



Coverage | New % | Master % | Delta
-------- | ----- | -------- | -----
**Canvas iOS** | 93.11% | 93.16% | -0.06%
Core/Core/SwiftUIViews/Icon.swift | 40% | -- | --
<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: scripts/build-icons.js [hunk 1]
```diff
@@ -177,41 +185,45 @@ for (const icon of whitelist) {
 }\n`)
 
     // Generate symbol image
-    continue // It's not ready yet
     run(`mkdir -p ${sfolder}`)
     fs.writeFileSync(`${sfolder}/${name}.svg`,
-`<svg version="1.1" xmlns="http://www.w3.org/2000/svg" viewBox="-192 -192 2304 2304">
+`
```
Comment (vanwagonet) [inline scripts/build-icons.js:190]: Is this blank first line intentional? It shows up in each svg file. You can put a `.trimStart()` at the end to eliminate it. I think we want the trailing newline though.

**Review by vanwagonet — CHANGES_REQUESTED** [review]
Looks great, just one nitpick.

File: scripts/build-icons.js [hunk 2]
```diff
@@ -177,41 +185,45 @@ for (const icon of whitelist) {
...
-  <rect id="left-margin" x="-192" y="-192" width="192" height="2304" />
-  <rect id="right-margin" x="1920" y="-192" width="192" height="2304" />
+  <line id="Baseline-S" x1="0" x2="24" y1="18" y2="18"/>
+  <line id="Capline-S"  x1="0" x2="24" y1="6" y2="6"/>
+  <line id="Baseline-M" x1="0" x2="24" y1="18" y2="18"/>
+  <line id="Capline-M"  x1="0" x2="24" y1="6" y2="6"/>
+  <line id="Baseline-L" x1="0" x2="24" y1="18" y2="18"/>
+  <line id="Capline-L"  x1="0" x2="24" y1="6" y2="6"/>
+  <line id="left-margin" x1="0" x2="0" y1="0" y2="24"/>
+  <line id="right-margin"  x1="24" x2="24" y1="0" y2="24"/>
```
Comment (vanwagonet) [inline scripts/build-icons.js:200]: looks like there's an extra space after the id here. if you are trying to line things up you actually want the extra space after left-margin.

**Review by vanwagonet — APPROVED** [review]

**Pull request merged**